docker-compose.integration-test.yml

```diff
@@ -2,23 +2,10 @@ version: "3"
 
 services:
   firefox:
-    image: b4handjr/selenium-firefox:python3-latest
-    env_file: .env
-    volumes:
-      - .:/code
-    links:
-      - nginx
-    expose:
-      - "4444"
-    shm_size: 2g
-    networks:
-      - private_nw
-      - public_nw
-  vnc:
     image: b4handjr/selenium-firefox:python3-latest
     env_file: .env
     environment:
-      - DISABLE_HEADLESS=1
+      - MOZ_HEADLESS
```

Comment:
Aha!!  I've been trying to figure out this pattern forever, I didn't know you do it like this.  That's awesome, thanks @jrbenny35 !!